components/sharing/SocialShareModal.vue

```diff
@@ -5,16 +5,46 @@
     hide-header-close
     hide-footer
     data-qa="share modal"
+    v-on="onItemPage && !oEmbedDataHtml?
+      { hide: resetEmbedCopied, show: requestEmbed}
+      :onItemPage ? { hide: resetEmbedCopied }
+        :{}"
   >
     <div class="icon-wrapper">
       <SocialShare
         :media-url="mediaUrl"
         with-text
       />
     </div>
+    <b-form
+      v-if="oEmbedDataHtml"
+      data-qa="share modal embed"
+      class="mt-3"
+      @submit.stop.prevent="submitForm"
+    >
+      <label> {{ $t('record.copyEmbedLabel') }}</label>
+      <b-form-textarea
+        id="shareEmbed"
+        ref="shareEmbed"
+        v-model="oEmbedDataHtml"
+        readonly
+        tabindex="0"
```

Comment:
Why is this needed?

Reply:
For accessibility we would want the textarea to be accessible through keyboard, since it's also clickable. But on reviewing, I guess we don't need to set tabindex, since textarea is an interactive element and already focusable. I believe that at some point I was testing and couldn't reach the textarea via keyboard so added the tabindex. Will take another look at this.

Reply:
Tested keyboard accessibility and it is indeed not needed to use tabindex.